applications/DEMApplication/python_scripts/DEM_analysis_stage.py

```diff
@@ -436,6 +436,7 @@ def InitializeTimeStep(self): # deprecated
         self.InitializeSolutionStep()
 
     def InitializeSolutionStep(self):
+        super(DEMAnalysisStage, self).InitializeSolutionStep()
         self._BeforeSolveOperations(self.time)
```

Comment:
Apparently, you have changed the order of things here. Before the PR, this `InitializeSolutionStep `was being executed after `_BeforeSolveOperations`.

Reply:
There was no call to super() before. It's there as required to move it out of the explicit_solver_strategy.cpp